docs/docs/getting-started/web.mdx

```diff
@@ -168,7 +168,9 @@ Some of these features are a work in progress, and some others will come later.
 
 **Coming soon**
 
-* `Font::GetPath`
+* `PathEffectFactory.MakeSum()`
+* `PathEffectFactory.MakeCompose()`
+* `Font.GetPath()`
 * `ShaderFilter`
```

Comment:
I think we should remove the coming soon section - We want to focus on what we have and keep our future plans "secret" :)

Reply:
Good point, I rephrased the section to be more sensible.